src/FormRequest.php

```diff
@@ -67,6 +76,7 @@ public function validate(): void
 
         $this->validator = $this->app->make('validator')
                                      ->make($this->all(), $this->rules(), $this->messages(), $this->attributes());
+        $this->withValidator();
```

Comment:
Move the validator make and with validator method to a different method as Laravel does. The method (ex: `getValidator`) will return a validator instance and that is set to the class property.

Example implementation:

```php
protected function getValidator(): Validator 
{
    $validator = $this->app
                                ->make('validator')
                                ->make($this->all(), $this->rules(), $this->messages(), $this->attributes());
    $this->withValidator($validator);

    return $validator;
}

public function validate()
{
   // ...
   $this->validator = $this->getValidator();
   // ...
}
```
https://github.com/laravel/framework/blob/ae0ef69bcc1573b4e6695bb7e560ded03eaa89ec/src/Illuminate/Foundation/Http/FormRequest.php#L81-L102